lib/screens/profile_screens/guardians/guardians_tabs/components/my_guardians_view.dart

```diff
@@ -43,10 +43,7 @@ class MyGuardiansView extends StatelessWidget {
                     }),
               )));
 
```

Comment:
never mind the comment

The bug was it was using "myGuardians.areGuardiansActive", not state.areGuardiansActive

I removed that method from guardian model. Model doesn't know if it's active.